src/main/webapp/autocomplete-test.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>shef</title>
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="script.js"></script>
+  </head>
+ 
+  <body onload="loadOptions();">
+    <input name="query" list="allOptions" />
```

Comment:
this doesn't really work with multiword queries. Do you have a plan to handle this? such as adding a new empty text box when the previous one has text entered.

Reply:
Changed to work w multiword queries.